app/javascript/dashboard/components/widgets/conversation/AssigneeTypeFilter.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+  <div>
+    <woot-button
+      variant="smooth"
+      size="small"
+      color-scheme="secondary"
+      class="selector-button"
+      @click="toggleDropdown"
+    >
+      {{ $t('CHAT_LIST.CHAT_ASSIGNEE_TYPE_FILTER_ITEMS')[activeType]['TEXT'] }}
+      <span class="assignee-type--count count">{{ activeTypeCount }}</span>
```

Comment:
Do we have to translate this to the current locale? `Number.toLocaleString`

Reply:
I think we are not using it anywhere. I will add it here.

Reply:
Done https://github.com/chatwoot/chatwoot/pull/6186/commits/ff39c184c70fbd6ea4db82e5f6767dbd9b2adb74